core/trino-main/src/main/java/io/trino/SystemSessionProperties.java

```diff
@@ -347,6 +348,11 @@ public SystemSessionProperties(
                         "Minimum data processed to trigger skewed partition rebalancing in local and remote exchange",
                         DataSize.of(200, MEGABYTE),
                         true),
+                integerProperty(
```

Comment:
`SCALE_WRITERS_MAX_SKEWED_PARTITION`.
Then locally we can derive: `max(SCALE_WRITERS_MAX_SKEWED_PARTITION, max_number_of_writers)`
Globally: `max(SCALE_WRITERS_MAX_SKEWED_PARTITION, numer_of_tasks)`